src/test/java/com/google/javascript/clutz/multifilePartial/singleton.d.ts

```diff
@@ -0,0 +1,44 @@
+//!! The singleton file is the result of a global compilation of the files in
+//!! the multifilePartial directory, so that the global and partial compilations
+//!! can be compared
```

Comment:
ok for now, but long term this file will be less and less useful (if we remove total mode one day). An even better test (in another PR) would a usage.ts file that typechecks together with the three partial .d.ts.

Reply:
That makes sense!